doc/src/sgml/mvcc.sgml

```diff
@@ -855,6 +856,12 @@ ERROR:  could not serialize access due to concurrent update
     until after the SQL standard was developed, and are outside the scope of
     this manual.  For a full treatment, please see
     <xref linkend="berenson95"/>.
+-->
+リピータブルリード分離レベルは、学術的なデータベースの文献や他のデータベース製品のいくつかでは<firstterm>スナップショット分離</firstterm>として知られる技術を用いて実装されています。
+同時実行性の面で劣る伝統的なロック技術を使うシステムと比較すると振舞いや性能の違いが観察されるかもしれません。
+他のシステムでは、リピータブルリードとスナップショット分離を異なる振舞いをする別の分離レベルとして提供しているかもしれません。
+2つの技術を区別する許容される現象は、標準SQLが制定されて以来データベース研究者により定式化されておらず、この文書の範囲を超えます。
```

Comment:
ここは、
「...標準SQLが制定されるまではデータベース研究者により定式化されておらず、この文書の範囲を超えます。」
ではないでしょうか。

Reply:
確認ありがとうございます。対応しました。
＃ 英文的にはご指摘の通りだと思うのですが、この文の内容に関しては残念ながら私の理解の**範囲を超えます**(T_T)

Reply:
遅レスですみません。
石井さんの指摘通りの訳でいいのではないか、という意見の補足です。

本修正があったMLを探していました。
https://www.postgresql.org/message-id/CA%2BhUKGJgSn%2BJF929bMpxhn21p0ArUwjs_RrxGKb2_6Fuu5J09w%40mail.gmail.com

この辺りです。
で、この話題で出てくる論文は以下ではないかと思います。

http://pmg.csail.mit.edu/papers/icde00.pdf

アブストラクトには以下のように記載があります。
Commercial databases support different isolation levels
to allow programmers to trade off consistency for a potential gain in performance. The isolation levels are defined in
the current ANSI standard, but the definitions are ambiguous and revised definitions proposed to correct the problem
are too constrained since they allow only pessimistic (locking) implementations. This paper presents new specifications for the ANSI levels.

それまで曖昧だったトランザクション分離レベルを改めて定義しました、
という内容のようで、ドキュメントが出たのが(様々なDBが出てからかなり年数の経った)2000年です。

なので、「標準SQLが制定されるまではデータベース研究者により定式化されておらず、」で良いと思います。

もしくは「標準SQLで制定されるまではデータベース研究者により定式化されておらず、」
などでしょうか。

Reply:
調査ありがとうございます。勉強になりました。